starter-code/index.js

```diff
@@ -0,0 +1,66 @@
+
+
+function deleteItem(){
+	var product = document.getElementById("product");
+	product.innerHTML="";
+}
+
+function getPriceByProduct(){
+	var precio = document.getElementsByClassName("price");
```

Comment:
Muy bien el approach solo recuerda que el cuando seleccionas por clase te regresa todos los que hay con esa clase. entonces debes acceder de esta forma:

 var precio = document.getElementsByClassName("price")[0].innerHTML.replace("$","");

y para sacar el de todos puedes hacerlo con un for